drivers/performance_measurement/app/models/performance_measurement/equity_analysis/builder.rb

```diff
@@ -0,0 +1,209 @@
+        send("describe_#{key}".to_sym)
+      end
+      param_rslt.push(describe_projects)
+      param_rslt.push(describe_project_type)
+      rslt = "#{rslt} <span>#{param_rslt.reject(&:blank?).join(', ')}</span>"
+      rslt
+    end
+
+    def chart_data
+      {
+        chart_height: @chart_data.chart_height,
+        data: @chart_data.data,
+      }
+    end
+
+    def show_additional_options?
+      metric.present? && investigate_by.present?
+    end
+
+    def metric
+      @params[:metric]&.to_sym
+    end
+
+    def describe_metric
+      metric_options.select { |_, sym| sym == metric }.first.first
+    end
+
+    def investigate_by
+      @params[:investigate_by]
+    end
+
+    def race
+      @params[:race]&.reject(&:blank?) || []
+    end
+
+    def describe_race
+      names = race.map do |key|
+        PerformanceMeasurement::EquityAnalysis::Data::RACES[key]
+      end.reject(&:blank?).join(', ')
+      race.any? ? "Race: #{names}" : ''
+    end
+
+    def age
+      @params[:age]&.reject(&:blank?) || []
+    end
+
+    def describe_age
+      names = PerformanceMeasurement::EquityAnalysis::Data::AGES.select { |_, v| age.map(&:to_sym).include?(v) }.keys.join(', ')
+      age.any? ? "Age: #{names}" : ''
+    end
+
+    def gender
+      @params[:gender]&.reject(&:blank?) || []
+    end
+
+    def describe_gender
+      names = gender.map do |id|
+        PerformanceMeasurement::EquityAnalysis::Data::GENDERS[id.to_i]
+      end.reject(&:blank?).join(', ')
+      gender.any? ? "Gender: #{names}" : ''
+    end
+
+    def household_type
+      @params[:household_type]&.reject(&:blank?) || []
+    end
+
+    def describe_household_type
+      names = household_type.map do |id|
+        PerformanceMeasurement::EquityAnalysis::Data::HOUSEHOLD_TYPES[id.to_i]
+      end.reject(&:blank?).join(', ')
+      household_type.any? ? "Household Type: #{names}" : ''
+    end
+
+    def project
+      @params[:project]&.reject(&:blank?) || []
+    end
+
+    def describe_projects
+      names = project.map do |d|
+        project_options.select { |o| o.last == d.to_i }.first&.first
+      end.reject(&:blank?).join(', ')
+      project.any? ? "Project: #{names}" : ''
+    end
+
+    def project_type
+      @params[:project_type]&.reject(&:blank?) || []
+    end
+
+    def describe_project_type
+      names = project_type.map do |d|
+        project_type_options.to_h.select { |_, v| v == d.to_i }.keys.first
+      end.reject(&:blank?).join(', ')
+      project_type.any? ? "Project Type: #{names}" : ''
+    end
+
+    def view_data_by
+      @params[:view_data_by] || VEIW_BY_DEFAULT
+    end
+
+    def metric_options
+      opts = []
+      @report.display_order.each do |sub_sections|
+        sub_sections.each do |keys|
+          keys.keys.each do |key|
+            next if key.in?(ignored_metrics)
+
+            opts.push([@report.detail_title_for(key), key])
```

Comment:
This does the same thing, but more closely matches syntax used elsewhere
```suggestion
            opts << [@report.detail_title_for(key), key]
```